build_library/generate_au_zip.py

```diff
@@ -277,17 +285,17 @@ def _ExcludeBlacklist(library_list, black_list=[]):
   return return_list
 
 
-def _EnforceWhiteList(library_list, white_list=[]):
-  """Deletes the set of files from black_list from the library_list
+def _EnforceAllowList(library_list, allow_list=[]):
+  """Deletes the set of files from deny_list from the library_list
     Args:
-      library_list: List of the library names to filter through black_list
-      black_list: List of the black listed names to filter
+      library_list: List of the library names to filter through deny_list
+      deny_list: List of the deny listed names to filter
     Returns:
       Filtered library_list
   """
```

Comment:
This whole doc string is a copy-pasta from `_ExcludeDenylist` and not even updated for `_EnforceAllowList`, should be more like (can't do a suggestion for the deleted lines, github tells me):
```python
  """Ensures that library_list contains all the items from allow_list
    Args:
      library_list: List of the library names to check
      allow_list: List of the items that ought to be in the library_list
  """
```

Reply:
Done